src/align.cpp

```diff
@@ -2072,7 +2067,7 @@ static void align_oc_msg_colon(chunk_t *so)
    {
       tmp = nas.m_aligned.GetChunk(idx);
 
-      tlen = tmp->str.size();
+      int tlen = tmp->str.size();
```

Comment:
@jibsen Yes, I had the same discussion with @gmaurel about deque<int>::size_type vs size_t, in the end we decided to use size_t. I hope at some point we `auto` everything anyway.